db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -107,26 +112,49 @@ def dialog_values(prov_option, options_hash, dialog_array)
 end
 
 def set_requested_value(prov_option, option_value, find_id, dialog_array)
-  $evm.log(:info, "set requested value: prov_option: #{prov_option} value: #{option_value}")
   return if cloud_value(prov_option, option_value, find_id, dialog_array)
 
   $evm.log(:info, "set requested value default_option: prov_option: #{prov_option} value: #{option_value}")
   default_option(option_value, dialog_array)
 end
 
+def cloud_storage(vmdb_object_find_by, dialog_array)
+  flavor = vmdb_object(:flavor, vmdb_object_find_by)
+  return false unless flavor
+  storage = flavor.root_disk_size.to_i + flavor.ephemeral_disk_size.to_i + flavor.swap_disk_size.to_i
+  default_option(storage, dialog_array)
+  true
+end
+
+def cloud_number_of_cpus(vmdb_object_find_by, dialog_array)
+  flavor = vmdb_object(:flavor, vmdb_object_find_by)
+  return false unless flavor
+  $evm.log(:info, "Retrieving cloud flavor #{flavor.name} cpus => #{flavor.cpus}")
+  default_option(flavor.cpus, dialog_array)
+  true
+end
+
+def cloud_vm_memory(vmdb_object_find_by, dialog_array)
+  flavor = vmdb_object(:flavor, vmdb_object_find_by)
```

Comment:
@tinaafitz 
Can we get the flavor object just once for the cloud and pass it around